mysite/announcements/models.py

```diff
@@ -0,0 +1,30 @@
+import datetime
+
+from django import forms
+from django.core.exceptions import ValidationError
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+from django.utils import timezone
+from django.contrib import admin
+from django.contrib.auth.models import User, Group
+
+@python_2_unicode_compatible
+class Announcements(models.Model):
+    title = models.CharField(max_length=32, help_text='This will be shown as the title of notification and announcement')
+    message = models.CharField(max_length=200, help_text='The message you want to convey')
+    # date_time_published = models.DateTimeField('datetime published', help_text='The date and time you want to schedule announcement/notification')
```

Comment:
Stray comments :/

It has been pointed out in previous review too to remove stray comments.

Reply:
Yes, counts stray wrt to this task.
For the new task, you should have a new branch.